plugins/techdocs/src/reader/components/Reader.tsx

```diff
@@ -89,11 +88,28 @@ export const Reader = () => {
       ]);
 
       divElement.shadowRoot.innerHTML = '';
+
       if (transformedElement) {
         divElement.shadowRoot.appendChild(transformedElement);
         transformer(divElement.shadowRoot.children[0], [
-          addEventListener({
-            onClick: navigate,
+          dom => {
+            setTimeout(() => {
```

Comment:
We had this exact same issue internally. The only problem is setTimeout isn’t a guarantee so we may end up having race conditions if the user’s internet connection isn’t good.

We can revisit this later in another PR in a more thoughtful way though